### DISCUSSION TITLE
Applied Forecasting

### DISCUSSION THREAD

**Discussion by varshasahasrabuddhe** [statmike/vertex-ai-mlops#42] [Q&A]
Hi Steve,
Do you have videos for Applied Forecasting?

Thanks,
Varsha

**Comment by statmike** [comment]
Hello @varshasahasrabuddhe ,
I have not done any new videos lately due to the large demand to create new content in the repository.  I do hope to get back to videos very soon. 

I am doing a full update for all the Applied Forecasting notebooks to include deeper descriptions, more methods, consistency across methods.  Check out the folder now and see if it is helpful.  I would love your feedback.  Also, to help me be motivated to do more videos let me know what you would like for me to include in the videos beyond what is in the notebooks!
Thank You,
Mike
